src/app/(with-header)/activities/[id]/components/ImageGrid.tsx

```
@@ -94,28 +105,55 @@ function ImageGrid({ mainImage, subImages }: ImageGridProps) {
 
       {/* PC/태블릿 */}
       <div className='hidden h-[500px] grid-cols-4 grid-rows-4 gap-6 md:grid'>
-        <div className='relative col-span-2 row-span-4 hover:animate-pulse'>
+        <div
+          onClick={() => handleImageClick(mainImage)}
+          className='relative col-span-2 row-span-4 hover:animate-pulse'
+        >
           <Image
-            src={mainImage}
+            src={image[0]}
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)